UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.API.FunctionalTests/Helpers/CommonHelper.cs

```diff
@@ -67,6 +68,20 @@ public static async Task RunWebJob()
                 await Task.Delay(70000);
             }
         }
+
+        public static async Task RunWebJobAio()
+        {
+            WebJob = new POSWebJob();
+            if (!posWebJob.IsRunningOnLocalMachine)
+            {
+                string POSWebJobUserCredentialsBytes = CommonHelper.GetBase64EncodedCredentials(posWebJob.UserName, posWebJob.Password);
+                POSWebJobApiResponse = await WebJob.POSWebJobEndPoint(aioWebJob.BaseUrl, POSWebJobUserCredentialsBytes);
+                POSWebJobApiResponse.StatusCode.Should().Be((HttpStatusCode)202);
+
+                //As there is no way to check if webjob execution is completed or not, we have added below delay to wait till the execution completes and files get downloaded.
+                await Task.Delay(70000);
```

Comment:
why a delay

Reply:
Hi Ray, we have added the comment just before the delay to summarize why we are using it. 
//As there is no way to check if webjob execution is completed or not, we have added below delay to wait till the execution completes and files get downloaded.
Thanks.